app/DoctrineMigrations/Version20211013095152.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20211013095152 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+	// this up() migration is auto-generated, please modify it to your needs
+	$this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+        $this->addSql('
+	   CREATE TABLE `sessions` (
+           `sess_id` VARBINARY(128) NOT NULL PRIMARY KEY,
+           `sess_data` BLOB NOT NULL,
+           `sess_lifetime` INTEGER UNSIGNED NOT NULL,
+           `sess_time` INTEGER UNSIGNED NOT NULL,
+           INDEX `sessions_sess_lifetime_idx` (`sess_lifetime`)
+	   ) COLLATE utf8mb4_bin, ENGINE = InnoDB;
+        ');
+
```

Comment:
Double nit, feel free to ignor:
1. Maybe indent the SQL 8 spaces to improve readability
2. Remove the linefeed on line 29

Reply:
:heavy_check_mark: